app/src/main/java/com/nutomic/syncthingandroid/activities/LogActivity.java

```diff
@@ -97,63 +98,80 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
-    private void scrollToBottom() {
-        mScrollView.post(() -> mScrollView.scrollTo(0, mLog.getBottom()));
-    }
-
     private void updateLog() {
-        if (mFetchLogTask != null)
+        if (mFetchLogTask != null) {
             mFetchLogTask.cancel(true);
+        }
```

Comment:
no return?

Reply:
I thought about this when making the PR. The original code intends to cancel and relaunch the asynctask, so no return here. I'll test this currently on my phone.